packages/tests/tests/peer_exchange.node.spec.ts

```diff
@@ -26,38 +26,41 @@ describe("Peer Exchange", () => {
       await waku?.stop();
     });
 
-    it("connection with fleet nodes", async function () {
-      // skipping in CI as this test demonstrates Peer Exchange working with the test fleet
-      // but not with locally run nwaku nodes
-      if (process.env.CI) {
```

Comment:
by doing so you are making a dependence of the js-waku CI run to be green on the wakuv2.test fleet.

I would usually advise against that. as you don't want developers to stop merging PR because an external system is down.

I would propose instead to run the live tests in a separate, non-required, CI pipeline.
The change would be a bit heavier, the idea would be to "only" run the live test in this pipeline.

Reply:
I agree with that, we definitely need another suit of test that is non-blocking: to that, I would move some of our async tests because they are sometimes not deterministic because of many await and event subscriptions. more complicated tests can be run on master/release branches

I slightly mentioned it here - https://github.com/waku-org/js-waku/issues/1430#issuecomment-1648582104

Reply:
added optional test step